### PR TITLE
PEN-1392 add social image metadata

### PR DESCRIPTION
**Pull request by kascote**
## Description
Updated MetaData component to add fallback site image as social image metadata. In the case the Authors page type, the image will be the author photo if exists.

## Jira Ticket
- [PEN-1392](https://arcpublishing.atlassian.net/browse/PEN-1392)

## Acceptance Criteria
As per this ALC document, image metadata should display for all page types, if it is present. It should be the Fallback Image set for the website. (If the fallback image is not present, then no image should display). https://corecomponents.arcpublishing.com/alc/arc-products/themes/user-docs/page-template-guide/page-types/

The image should appear in both twitter:image and og:image meta tags

## Test Steps
- check on `/pf/tags/beaches` the image and alt:image used is the fallback one
- check on `/pf/section/news` the image and alt:image used is the fallback one
- check on `/pf/author/alexandra-petri` the image and alt:image used the one from the author
- check on `/pf/homepage` the image and alt:image used is the fallback one

## Dependencies or Side Effects
- 

## Review Checklist
- [x] Confirmed all the test steps above are working
- [x] Confirmed there are no linter errors
- [x] Confirmed this PR has reasonable code coverage
  - [x] Confirmed this PR has unit test files
  - [x] Ran `npm test`, made sure all tests are passing
  - [ ] If the amount of work to write unit tests for this change are excessive,
please explain why (so that we can fix it whenever it gets refactored).
- [ ] Confirmed relevant documentation has been updated/added.

## NOTE

I bypassed the pre-commit hook because errors on the chromatic component. I executed `jest`and `lint` and all green before generate this PR.

Screen Shot 2020-11-04 at 15 53 36

### CODE REVIEW

**Comment by JackHowa** [comment]
@kascote if you pull in new canary, you shouldn't see that storybook error https://github.com/WPMedia/engine-theme-sdk/pull/114

File: src/components/MetaData/index.test.tsx [hunk 1]
```diff
@@ -140,6 +143,7 @@ const wrapperGenerator = (
       arcSite={arcSite}
       websiteDomain={websiteDomain}
       facebookAdmins={facebookAdmins}
+      fallbackImage={localFallback ? fallbackImageLocal : fallbackImageRemote}
```
Comment (JackHowa) [inline src/components/MetaData/index.test.tsx]: I'm kind of confused by this test. how does <MetaData /> know whether an image is a local image or remote?

**Comment by JackHowa** [comment]
Also, I'm looking for a test if there's either no fallbackImage. Potentially the field could be empty, I think

File: src/components/MetaData/index.test.tsx (hunk 1 above)
Comment (kascote) [reply src/components/MetaData/index.test.tsx]: by the parameter `localFallback`, the test will send what to test

**Comment by kascote** [comment]
I'm adding that test case

**Review by JackHowa — APPROVED** [review]
Thanks Nelson. I'll publish this to engine theme sdk and merge the metadata block

**Pull request merged**